src/components/member-profile/index.js

```diff
@@ -358,7 +362,8 @@ const Profile = (props) => {
           <div className={classNames.memberDetails}>
             <motion.img
               layoutId={username}
-              src={imageLink.w_200 ?? '/icons/user.png'}
+              src={imageLink.w_200 ?? '/images/Avatar.png'}
```

Comment:
What if src has the wrong image URL(not null or undefined) which is not even existing? will it go for `images/Avatar.png`?

Reply:
Yes. If it's not able to fetch the member's image, it will go for Avatars.png. I have tested it for every scenario.

Reply:
Testing every scenario is really great skill as a developer. But since we are already using the onError method, which will take care of all such scenarios.

Reply:
Have you checked without the onError method?

Reply:
The `onError` method won't work if the user has given `src` as `imageLink.w_500`. It will produce the following output:

![image](https://user-images.githubusercontent.com/35677839/148644887-7ce03476-4dfb-4d46-8531-9b0f5f5fe923.png)


`onError` method will work in the following scenarios:
1. when `src === ""`
2. when `src === {imageLink.w_200}`

- But if we alter the numbers inside `imageLink`, the output shown above will be produced which we clearly don't want.
   May be this scenario was ignored. 
- `onError` method will come into effect for all other cases except this one. That's why we need to handle the `src` 
   attribute too.